### PR TITLE
Apply minor fixes to Marketplace page and README

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: DESCRIPTION.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Probably the sharpest git repository organizer and rebase/merge workflow automation tool you've ever seen!
```
Comment (PawelLipski) [inline DESCRIPTION.html:1]: **TODO for Marketplace**: I don't think the static image we have there:

image

is the most fortunate choice (too many details, yellow edge etc.). My suggestion is to replace it with docs/marketplace/sample_status.png - WDYT :question:

Comment (mkondratek) [reply DESCRIPTION.html:1]: sounds good, agreed (sounds like Hagrid :Xd:)

File: README.md [hunk 2]
```diff
@@ -6,18 +6,18 @@
 
 <img src="src/main/resources/META-INF/pluginIcon.svg" width="130" height="130" align="left"/>
 
-Git Machete plugin is a robust tool that simplifies your git related workflow.
+Git Machete plugin is a robust tool that **simplifies your git related workflow**.
 It's a great complement to the JetBrains products' built-in version control system.<br/>
 
-The "bird's eye view" provided by Git Machete makes merges/rebases/push/pulls hassle-free
-even when multiple branches are present in the repository
+The _bird's eye view_ provided by Git Machete makes **merges/rebases/push/pulls hassle-free**
```
Comment (mkondratek) [inline README.md:12]: These changes brought a spirit to this part :+1:

File: build.gradle [hunk 3]
```diff
@@ -469,6 +469,8 @@ patchPluginXml {
   // In `untilBuild` situation is inverted: it's inclusive when using `*` but exclusive when without `*`
   untilBuild intellijVersionToBuildNumber(intellijVersions.current) + '.*'
 
+  // Note that the first line of the description should be self-contained since it is placed into embeddable card:
```
Comment (mkondratek) [inline build.gradle:472]: :rofl:

**Review by mkondratek — APPROVED** [review]

**Review by micpiotrowski — APPROVED** [review]

File: DESCRIPTION.html (hunk 1 above)
Comment (PawelLipski) [reply DESCRIPTION.html]: Done

**Comment by PawelLipski** [comment]
@micpiotrowski @mkondratek a bunch of new fixes pushed

File: DESCRIPTION.html [hunk 4]
```diff
@@ -1,13 +1,16 @@
-Probably the sharpest git repository organizer and rebase/merge workflow automation tool you've ever seen!
+Probably the sharpest <b>git repository organizer</b> and <b>rebase/merge workflow automation</b> tool you've ever seen! <br/><br/>
```
Comment (mkondratek) [inline DESCRIPTION.html:1]: Will it work?

**Review by mkondratek — APPROVED** [review]

File: DESCRIPTION.html [hunk 5]
```diff
@@ -1,11 +1,16 @@
-This plugin adds an extra tab in the Git tool window that gives the instant answer to the questions:
-<i>what are the branches in this repository? what needs be merged/rebased/pushed/pulled?</i> <br/><br/>
+Probably the sharpest <b>git repository organizer</b> and <b>rebase/merge workflow automation</b> tool you've ever seen! <br/><br/>
 
-The bird's eye view provided by Git Machete makes rebases/push/pulls hassle-free
-even when multiple branches are present in the repository:
+This plugin adds an extra tab in the Git tool window that gives an instant answer to the questions:
+<ul>
+<li>what are the branches in this repository?</li>
+<li>what needs be merged/rebased/pushed/pulled?</li>
```
Comment (mkondratek) [inline DESCRIPTION.html:6]: What about starting these with the capital `W`?

**Review by micpiotrowski — APPROVED** [review]

File: DESCRIPTION.html (hunk 4 above)
Comment (PawelLipski) [reply DESCRIPTION.html:1]: Not sure... let's give it a shot. `<b>` works in `General Usage Instructions` and `<i>` works here in the description.

**Pull request merged**